pkg/health/health_test.go

```diff
@@ -27,7 +27,27 @@ func recordCall(ts *httptest.Server, path string) *http.Response {
 	return w.Result()
 }
 
-var _ = Describe("Test httpserver", func() {
+var _ = Describe("test health probe helpers", func() {
```

Comment:
Can we convert these to use Go's `testing` pkg instead? A lot of times I have seen developers rip Ginkgo tests later anyway. I suggest sticking to Go `testing` lib for new tests.

Reply:
I am adding to existing tests.  Not going to have much bandwidth to convert.